ddprof-lib/src/main/cpp/profiler.cpp

```diff
@@ -906,14 +897,30 @@ void Profiler::busHandler(int signo, siginfo_t *siginfo, void *ucontext) {
 }
 
 bool Profiler::crashHandler(int signo, siginfo_t *siginfo, void *ucontext) {
+  ProfiledThread* thrd = ProfiledThread::current();
+  if (thrd != nullptr && !thrd->enterCrashHandler()) {
```

Comment:
NIT: you would usually have a RAII object here, to just call exit crashtracker on object destruction.

Reply:
the `longjmp` makes this hard, got it